test/remote/gateways/remote_sage_pay_test.rb

```diff
@@ -159,12 +159,13 @@ def test_successful_visa_purchase
     assert !response.authorization.blank?
   end
 
-  def test_successful_maestro_purchase
-    assert response = @gateway.purchase(@amount, @maestro, @options)
-    assert_success response
-    assert response.test?
-    assert !response.authorization.blank?
-  end
+  # Maestro is not available for GBP
+  # def test_successful_maestro_purchase
+  #   assert response = @gateway.purchase(@amount, @maestro, @options)
+  #   assert_success response
+  #   assert response.test?
+  #   assert !response.authorization.blank?
+  # end
```

Comment:
@davidsantoso Theoretically it works and is needed for other test accounts not using GBP, so I thought I'd leave it in. I'm fine with uncommenting it and having one failing test, though.